flat/lib/react.js

```diff
@@ -22,7 +23,7 @@ module.exports = function react() {
       plugins: {
         react: reactPlugin,
         "react-hooks": hooksPlugin,
-        "jsx-a11y": jsxA11yPlugin,
+        "jsx-a11y": fixupPluginRules(jsxA11yPlugin),
```

Comment:
![ask](https://img.shields.io/badge/review-ask-blue.svg)
Is `fixupPluginRules` necessary?

Reply:
Indeed! I used to use it for temporary v9 support, but that is no longer necessary!
I'll delete it!